improver_tests/blending/weighted_blend/test_WeightedBlendAcrossWholeDimension.py

```diff
@@ -209,15 +209,9 @@ class Test_check_percentile_coord(Test_weighted_blend):
 
     def test_basic(self):
         """Tests the basic use of check_percentile_coord"""
-        expected = DimCoord(
-            np.array([0.0, 20.0], dtype="float32"),
-            standard_name=None,
-            units="%",
-            long_name="percentile",
-        )
         cube = self.perc_cube[:2]
-        x = WeightedBlendAcrossWholeDimension.check_percentile_coord(cube)
-        self.assertEqual(x, expected)
+        has_perc = WeightedBlendAcrossWholeDimension.check_percentile_coord(cube)
+        self.assertTrue(has_perc)
```

Comment:
I think I'd prefer a single statement here:
```suggestion
        self.assertTrue(
            WeightedBlendAcrossWholeDimension.check_percentile_coord(cube)
        )
```